try raise(RuntimeException) catch RuntimeException as e in raise(e)